### ISSUE TITLE
VGG16 convergence comparison on GPU between fluid and tensorflow

### ISSUE THREAD

**Issue by kuke** [PaddlePaddle/Paddle#6813]
![vgg convergence](https://user-images.githubusercontent.com/3064195/34239204-be23d556-e640-11e7-8f77-68715a8cdd61.png)

- Data set: cifar10
- Epochs: 50
- Batch size: 128 
- Optimizer: Adam
- Learning rate = 1e-3
- Conclusion: Fluid converges a bit faster and they get very close accuracy after adequate training
